### DISCUSSION TITLE
Campaign Indiegogo either Kickstarter

### DISCUSSION THREAD

**Discussion by Corsair-63** [peterantypas/maiana#29] [Ideas]
Maybe should be a good option to star a campaign in one of those platforms in order to get funds and pre-order, orders of this system.

**Comment by mickymills** [comment]
I’m interested in this

**Reply by Pagandy** [reply]
Me too.

**Reply by gardner** [reply]
How many units would we need to sell to access economy of scale type manufacturing?

**Reply by peterantypas** [reply]
I'm assuming that you're referring to the entire system, not just the PCBAs, correct?

**Reply by peterantypas** [reply]
There are a few big challenges to overcome in order for MAIANA to be eligible for mass production:

1. Regulatory status. With its Open Source nature, it is extremely unlikely that it can receive an FCC grant as an intentional radiator. The FCC doesn't like it when an end user can modify device firmware and thus its operating frequency. That's why many WiFi drivers in the Linux kernel are distributed as .o files (compiled binary) rather than .c (source). It's not strictly for IP protection; it's the law.
2. It's intentionally disregarding some aspects of the AIS standard (such as DSC scanning) so that it can remain low-cost. Nobody makes silicon for decoding audio modulation anymore, especially something as niche and arcane as DSC. We would have to obtain a waiver from the Coast Guard for this.
3. The Open Source nature flies in the face of another Coast Guard rule re: provisioning. A supplier is required to preprogram every AIS device they ship to US customers and the end user should not have the capability to modify its static parameters (MMSI, name, call sign, etc). 

Only after a clear legal path emerges to address all of the above can we begin considering "economies of scale". I'm wide open to suggestions :)

**Reply by gardner** [reply]
That is really frustrating. I didn't realise those agencies make things so difficult. That Coast Guard rule must hinder the secondary market for AIS in the USA.

As not a lawyer: the simplest legal path that I can think of would be to not take orders/ship to the USA. 😂 

For DSC scanning, I am naive to the complexities of the standard. Some handheld VHFs have AIS with DSC. Perhaps this is just sending DSC, not scanning for it. Would incorporating something like the CMX7032/CMX7042 address that problem?

It is probably a huge headache but getting FCC certification and working to get Coast Guard buy-in would be one way to get to market. The thing is, the product that you made is really unique and cool. The form factor / all-in-one completeness makes it special. The cheapest transceiver on West Marine's website is $750 and that's a box that gets bolted on somewhere down below.

**Reply by muebau** [reply]
Just to understand the problem:

What is the deal with DSC? Its digital encoded information like AIS is to, right? It should be no problem to decode it from channel 70, am I wrong?

**Reply by muebau** [reply]
As I found it's FSK @ 156.525 MHz with 1200 baud, 10 bit with error correction.
Given that AIS is GMSK with 9600 bit/s DSC looks even much simpler to me.
Sure an additional transceiver would be needed just for DSC. The DSC part is just required for distress relay I guess.

**Reply by peterantypas** [reply]
DSC is used to broadcast frequency management messages at different times of day. AIS devices only receive those. So the difficulty there is locating those signals in the first place. But, you may be right, it may be simpler than I think.

That said, the whole point is to detect if AIS has been switched away from the two default channels. This can simply be done by having the main receivers scan the entire band at startup. 99.99999999999% of the time the traffic will be on the default channels. No authority regularly changes AIS frequencies AFAIK, and the USCG did it only once by accident, causing all kinds of confusion.

**Comment by gah67** [comment]
The problem is, every country also has it's own regulatory systems. 
An AIS system in the UK/EU has to conform with the European Radio Equipment and Telecommunications Terminal Equipment (R&TTE) Directive (99/5/EC) and the Marine Equipment Directive MED (96/98/EC). 
Separately, in the UK it also has to meet the relevant UK Radio Interface Requirement (IR) and I expect it also has to pass tests or be licenced by OfCom as well. 

The equipment ALSO has to pass CE testing which is expensive & time consuming & I don't think in it's current form would pass 

Now you know why commercial equipment is so expensive, it's not the contents of the box you are paying for it's the R&D & regulatory costs - oh and they may the boxes nice and big as that's what consumers like when they pay out that sort of cash :)

**Comment by marcelrv** [comment]
You buy the unlicensed AIS boxes already from $200 on aliexpress (incl
shipping). Hence supporting the comment of @gah67
Hence even lower priced than the MAIANA (in Europe). MAIANA has the benefit
of the formfactor & OSS

**Comment by muebau** [comment]
In Europe the RED will change the game even more:
https://ec.europa.eu/growth/sectors/electrical-engineering/red-directive_en

TLDR; I will be forbidden to have devices on the market with the direct or indirect ability to change any of the radio (software) part most likely.

I am very happy to see forward to MAIANA as it will assure my idea of a "free" ship.

**Comment by Rudder2** [comment]
Wow, a mandate because of an accidental event. That's so government...

**Comment by peterantypas** [comment]
So, in sum, MAIANA will remain the AIS system for the free people of this world. It will never become commercial. I intend to publish everything on Github, including the antenna design & BOM, as well as the FreeCAD files for all the plastic parts.

**Comment by Rudder2** [comment]
I totally agree with that. The open source nature of MAIANA is why I like
it and I accept those terms. Thank you for all the work on this open source
project.

**Comment by peterantypas** [comment]
All of the problems I mentioned stem from the fact that MAIANA is a transponder. Now, if it was instead marketed as a receiver and by means of the grapevine (i.e. Google) one could find and procure a solder-on module that would turn it into a transponder ... and if said component was also open source ...

**Comment by Pagandy** [comment]
Or even just a shunt...

**Reply by peterantypas** [reply]
... or a solder bridge ....

**Reply by peterantypas** [reply]
In all seriousness, though, there has to be sufficient "engineering" effort to qualify as a home-built device. Assembly alone does not cut it. But I believe that's where the solution lies.

**Comment by gah67** [comment]
In the UK & EU at least it could not be sold as a complete item without CE testing & certification

**Comment by peterantypas** [comment]
A plan has emerged. It's now a matter of finding the right fulfillment partner in Hong Kong. Here is the outline:

1. MAIANA's next main board does not contain a transmitter circuit, but has a footprint for a module that contains the transmitter
2. If someone wants to order the kit, they can still do it but it ships from here without the transmitter (for a reduced price, of course)
3. If they want to enable transmission, they order the transmitter module from an outfit in HK, and they simply solder it onto the main board with an iron (it will be that simple). The cost of the module will be something really low like $5 + shipping (which should be relatively cheap)

The main benefit of this approach is that I can legally ship everything else without issues, so I can actually go "public" with this rather than operate under the radar. Everything remains Open Source, of course (all designs will be on Github), so if folks want to build things on their own, they can. 

The overall cost of the system will still be a fraction of the commercial solutions, and without fear of legal retribution I can achieve economies of scale on the rest of the stuff without compromising its quality. 

The downside, of course, is that everyone will have to do a bit more work on their own, but I don't think that's unreasonable.

**Reply by peterantypas** [reply]
And here it is:

![maiana-receiver-1 0](https://user-images.githubusercontent.com/1565933/126908050-be9b4d95-1624-49b5-8613-b390e2d4f5d4.png)

![tx-module-1 0](https://user-images.githubusercontent.com/1565933/126908054-df1aaffa-6b02-45a9-9bc1-0c3548f036be.png)

**Comment by Pagandy** [comment]
Can't wait to get soldering.

Best Regards

Andy Pag

--
Need more?
@andybutshorter
BTC: 3JmkFBF6Y26Kp9j49DjQSf5vcncJJfrDmT
--
